src/components/detailedView/modelView/modelView.tsx

```diff
@@ -1,114 +1,113 @@
-import { Classes } from "@blueprintjs/core";
+import { Classes, Tab, Tabs } from "@blueprintjs/core";
 import "@blueprintjs/core/lib/css/blueprint.css";
 import axios from "axios";
 import * as React from "react";
+import { API_ROOT } from "../../../api-config";
+import PropertiesView from "../propertiesView";
 import { RouteComponentProps, withRouter } from "react-router";
+import { connect } from "react-redux";
+import { ModelObject } from "../../utils";
 
-export interface ModelViewProps { rid: string };
+export interface ModelViewProps { token: string, rid: any };
 
 interface ModelViewState {
-  comment: any,
-  cpu: any,
-  display_color: any,
-  height: any,
-  memory_gb: any,
-  model_id: any,
-  model_number: any,
-  num_ethernet_ports: any,
-  num_power_ports: any,
-  storage: any,
-  vendor: any
+  instances: Array<any> | undefined,
+  model: ModelObject | undefined,
+  columns: Array<string>,
+  fields: Array<string>
 }
 
-async function getData(modelkey: string) {
+async function getData(modelkey: string, token: string) {
+  console.log( API_ROOT + "api/models/" + modelkey)
+  const headers = {
+    headers: {
+      Authorization: "Token " + token
+    }
+  }
   return await axios
-    .get("https://rack-city-dev.herokuapp.com/api/models/" + modelkey)
+    .get(API_ROOT + "api/models/" + modelkey, headers)
     .then(res => {
       const data = res.data;
       return data;
     });
 }
 
-export class modelView extends React.PureComponent<RouteComponentProps, ModelViewState> {
+export class modelView extends React.PureComponent<RouteComponentProps & ModelViewProps, ModelViewState> {
 
   public state: ModelViewState = {
-    comment: "",
-    cpu: "",
-    display_color: "",
-    height: "",
-    memory_gb: "",
-    model_id: "",
-    model_number: "",
-    num_ethernet_ports: "",
-    num_power_ports: "",
-    storage: "",
-    vendor: ""
-  }
-
-  async componentDidMount() {
-    const resp = await getData("1");
-    this.setState({
-      comment: resp.comment,
-      cpu: resp.cpu,
-      display_color: resp.display_color,
-      height: resp.height,
-      memory_gb: resp.memory_gb,
-      model_id: resp.model_id,
-      model_number: resp.model_number,
-      num_ethernet_ports: resp.num_ethernet_ports,
-      num_power_ports: resp.num_power_ports,
-      storage: resp.storage,
-      vendor: resp.vendor
-    })
+    instances: undefined,
+    model: undefined,
+    columns: ["Model #", "CPU", "Height", "Display Color", "Memory (GB)", "# Ethernet Ports",
```

Comment:
we should probably decide on a a standard property display name and try to standardize this across the propertyviews and tables (i think this looks good)